test-utils/domMock.js

```diff
@@ -79,7 +79,26 @@ module.exports = function() {
 	var $window = {
 		document: {
 			createElement: function(tag, is) {
+				var cssText = ""
 				var style = {}
+				Object.defineProperty(style, "cssText", {
+					get: function() {return cssText},
+					set: function (value) {
+						if (typeof value === "string") {
+							for (var key in style) style[key] = ""
+							var rules = value.split(";")
```

Comment:
Aside, I kept this from the original implementation, but it will break if a semicolon is passed in a string value 

`foo: ";"; bar: 5`

@lhorie do you want it fixed?